graph.h

```diff
@@ -17,8 +17,7 @@ struct node {
 
 	/* generating edge and dependent edges.
 	 *
-	 * only gen and nuse are set in parse.c:parseedge; use is allocated and
-	 * populated in build.c:computedirty. */
+	 * gen and use are set in parse.c:parseedge */
```

Comment:
I think you can just remove this part of the comment completely. Now that `gen` and `use` are set during edge construction with everything else, there is no need to clarify here.